bpf/lib/conntrack_map.h

```diff
@@ -49,6 +49,7 @@ struct {
 		__type(key, struct ipv6_ct_tuple);
 		__type(value, struct ct_entry);
 		__uint(max_entries, CT_MAP_SIZE_TCP);
+		__uint(map_flags, LRU_MEM_FLAVOR);
```

Comment:
We may want a release note that if you had distributed LRU enabled previously, you may see disruptions during an upgrade as may force some maps to be recreated?

Reply:
Will tweak the release note a bit, agree.